locales/en.js

```diff
@@ -0,0 +1,187 @@
+export const en = {
+  welcome: {
+    title: 'Open your new bank account',
+    description_p_1: 'To open a bank account, we will need to verify your identity.',
+    description_p_2: 'It will only take a couple of minutes.',
+    next_button: 'Verify Identity'
+  },
+  document_selector: {
+    title: 'Verify your identity',
+    hint: 'Select the type of document you would like to upload',
+  },
+  upload_desktop: {
```

Comment:
there seem to be redundant strings. for sure something we need to avoid, otherwise with more languages it will become very hard to manage. Denesting might be preferable as well, since that way we can decouple logic from strings as much as possible.

Reply:
there are some techniques that Android uses for this scenario (desktop vs mobile) that we could mimic